index.js

```diff
@@ -1,106 +1,151 @@
 const path = require("path");
 const { execSync } = require("child_process");
 const fs = require("fs");
-const axios = require('axios');
-const FormData = require('form-data');
-const converter = require('json-2-csv');
-const split = require('split');
-const { finished } = require('stream');
+const axios = require("axios");
+const FormData = require("form-data");
+const converter = require("json-2-csv");
+const split = require("split");
+const { finished } = require("stream");
 // Constructing promisify from util
-const { promisify } = require('util');
+const { promisify } = require("util");
 // Defining finishedAsync method
 const finishedAsync = promisify(finished);
 
+const eventEndTimeLimit = 5 * 60 * 1000; // after this time limit, events are sent even if not ended
```

Comment:
EVENT_ENT_TIME_LIMIT

Reply:
I think that it would be better approach to pass this params as part of config and then read in initialize, if we miss some param then we use default constant that we defined locally. 

In that way we can change params like file names, end time from ot-node if we won't, no need to always update plugin and publish it on npm.